pxtlib/docsrender.ts

```diff
@@ -275,10 +278,17 @@ namespace pxt.docs {
             params["github"] = "";
         }
 
+        // Add accessiblity menu 
+        const accMenuHtml = `
+            <a href="#maincontent" class="ui item link" tabindex="0" role="menuitem">${lf("Skip to main content")}</a>
```

Comment:
What is "main content" supposed to be? Aren't they skipping to the JavaScript editor?